oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -105,6 +105,91 @@ def get_computer_system(uuid):
         return abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+@computer_system.route("/redfish/v1/Systems/<uuid>/"
+                       "Actions/ComputerSystem.Reset", methods=["POST"])
+def change_power_state(uuid):
+    """Change the Oneview power state for a specific Server hardware.
+
+        Return ResetType Computer System redfish JSON for a
+        given server hardware UUID.
+        Logs exception of any error and return abort.
+
+        Returns:
+            JSON: Redfish JSON with ComputerSystem ResetType.
+
+        Exceptions:
+            HPOneViewException: When some OneView resource was not found.
+            return abort(404)
+
+            OneViewRedfishError: When occur a power state mapping error.
+            return abort(400)
+
+            Exception: Unexpected error.
+            return abort(500)
+    """
+
+    try:
+        try:
+            reset_type = request.form["ResetType"]
```

Comment:
this line is supposing we are using a "form" type. According to the spec (http://redfish.dmtf.org/schemas/DSP0266_1.1.html#actions-property) is must be a application/json payload. I think it is available at `request.json["ResetType"]`